project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject puppetlabs/lein-ezbake "1.4.1-SNAPSHOT"
+(defproject puppetlabs/lein-ezbake "1.5.0-SNAPSHOT"
```

Comment:
And another example: https://github.com/puppetlabs/ezbake/commit/72ece8d1ab8481b9f4925714f33a031534e57dc4